doc/developers-guide/Testing-MongooseIM.md

```diff
@@ -200,11 +199,35 @@ Altogether, we have eight preset configuration.
 
 If you want to dig deeper, consult `.travis.yml` and `tools/travis-test.sh`, everything we do is there.
 
+### Gathering test reports from Travis tests
+
+If you test your MongooseIM fork on Travis, you might want to access test reports (which also include node logs and crash dumps) that are created by the test runner.
+
+#### Uploading reports to S3
+
+Our script uses AWS CLI to upload test results to an S3 bucket.
+Simply set [relevant environment variables](https://docs.aws.amazon.com/cli/latest/userguide/cli-environment.html) in your repository settings on Travis (at least `AWS_ACCESS_KEY_ID` and `AWS_SECRET_ACCESS_KEY` have to be set), and enjoy test reports landing straight into your bucket (`AWS_BUCKET` variable should store the bucket's name).
+
+#### Uploading reports to Google Drive
+
+To store test results in Google Drive you need to [create a new project and obtain service account credentials](https://developers.google.com/identity/protocols/OAuth2ServiceAccount).
+You must also add Google Drive API to your project - to do this, navigate to *APIs & Services* in your project console and find & add *Google Drive API* in the *Library* tab.
+Once downloaded, encode the credentials file with base64 (e.g. `cat serviceCreds.json | base64`) and use the result as `GDRIVE_SERVICE_ACCOUNT_CREDENTIALS` environment variable in your Travis repository settings.
```

Comment:
can we have an example of how serviceCreds.json looks like?

Reply:
Link to here can be useful too
https://github.com/prasmussen/gdrive#service-account

Reply:
It doesn't matter what it looks like though. AWS could change the format tomorrow and the upload script would still work (but our documentation would be outdated).